docs/workflow-and-job.md

```diff
@@ -10,151 +10,222 @@ It can execute anything that can be executed from the Python code, for example:
 * BigQuery query
 * Any Python code.
 
-The simplest workflow you can create, looks like this:
+The simplest workflow you can create looks like this:
 
+[`./docs_examples/workflow_and_job/simple_workflow_and_job.py`](./docs_examples/workflow_and_job/simple_workflow_and_job.py)
 ```python
 from bigflow.workflow import Workflow
 
-class HelloWorldJob:
+class SimpleJob:
     def __init__(self):
-        self.id = docs
+        self.id = 'simple_job'
 
     def run(self, runtime):
-        print(f'Hello world at {runtime}!')
+        print(f'Running a simple job')
 
-hello_world_workflow = Workflow(workflow_id='hello_world_workflow', definition=[HelloWorldJob()])
+simple_workflow = Workflow(workflow_id='simple_workflow', definition=[SimpleJob()])
 ```
 
-Now, you can run the `hello_world_workflow`:
+You can run the workflow and job within a module (but we recommend using the BiggerQuery CLI):
```

Comment:
if we we recommend using BiggerQuery CLI, why it's not shown here?

Reply:
done